perf_tests/CMakeLists.txt

```
@@ -24,8 +35,10 @@ add_executable(perf_test-main test_main.cpp
   test_sendrecv.cpp
   test_2dhalo.cpp
 )
-kokkoscomm_add_cxx_flags(TARGET perf_test-main)
-target_link_libraries(perf_test-main KokkosComm benchmark::benchmark)
+if(KOKKOSCOMM_TESTS_ARE_SUBDIRECTORY)
```

```suggestion 
get_directory_property(hasParent PARENT_DIRECTORY)
if(hasParent)
```
or just
```suggestion
if(KOKKOSCOMM_ENABLE_TESTS)
```
or again
```suggestion
if (NOT TARGET KokkosComm::KokkosComm)
```
?